.github/workflows/sbt-tasks.yml

```diff
@@ -0,0 +1,71 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: SBT Tasks
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up JDK 11
+      uses: actions/setup-java@v4
+      with:
+        java-version: '11'
+        distribution: 'temurin'
+
+    - name: Run tests
+      run: build/sbt test
+    - name: Run license check
+      run: build/sbt licenseCheck
+    - name: Run tests with coverage
+      run: build/sbt jacoco
```

Comment:
does this run  all the tests all of over again? If not, then maybe we should rename this to "Generate coverage". If yes, then we should just ensure that tests run only once.